src/test/java/com/cdancy/bitbucket/rest/features/HookApiLiveTest.java

```diff
@@ -27,11 +27,13 @@
 import com.cdancy.bitbucket.rest.domain.repository.HookSettings;
 import com.cdancy.bitbucket.rest.domain.repository.Repository;
 import com.cdancy.bitbucket.rest.domain.repository.RepositoryPage;
-import com.google.gson.internal.LinkedTreeMap;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
:warning: Wrong lexicographical order for 'java.util.Map' import. Should be before 'org.testng.annotations.Test'. (#QR132:35#)-CHECKSTYLE_CustomImportOrderCheck